### PR TITLE
Show 6 participants in a row insted of 6 issue::117

### LINKED ISSUE #117: Show 6 participants in a row instead of 4.

### PR DESCRIPTION
**Pull request by itsnkm2000**
fixes #117

### CODE REVIEW

**Comment by prateek76** [comment]
@itsnkm2000 update description with issue number and provide a live link

**Comment by itsnkm2000** [comment]
https://itsnkm2000.github.io/Opencode-Collaborative-19/.
live link updated

File: js/main.js [hunk 1]
```diff
@@ -58,7 +58,7 @@ $(document).ready(function () {
                 } catch (err) {}
             }
             var mentorDiv =
-                "<div class='col-lg-3 col-sm-6 text-center mb-4'>" +
+                "<div class='col-lg-2 col-sm-6 text-center mb-4'>" +
```
Comment (pradeepgangwar) [inline js/main.js]: You were supposed to change participants card not mentors.

**Comment by ananya-agrawal** [comment]
Closing pr due to inactivity

**Pull request closed without merging**